terraform/pif.gov.tf

```diff
@@ -127,22 +127,38 @@ resource "aws_route53_record" "fellows_in_innovation_pif_cname" {
   records = ["d3at1jdwnpqw7w.cloudfront.net"]
 }
 
-resource "aws_route53_record" "paygap_pif_cname" {
+resource "aws_route53_record" "d_18f_gov__acme_challenge_paygap_pif_cname" {
```

Comment:
Since this is a `pif.gov` top level domain, should we set the resource the name to be something like `d_pif_gov_...`